src/ccnl-core/src/ccnl-prefix.c

```diff
@@ -671,11 +671,12 @@ ccnl_prefix_to_str_detailed(struct ccnl_prefix_s *pr, int ccntlv_skip, int escap
                             || (escape_components && c == '/' )) ?
```

Comment:
This is clearly not part of the scope, but 0x20 and 0x7F refers to ASCII values (space and del character)?